scripts/process-service.js

```diff
@@ -8,8 +8,17 @@ const { unique } = require('./util/set');
 
 const outRoot = sysPath.resolve(`${__dirname}/../src/AWS`);
 
+const findIOShapes = (ops, io) =>
+  Object.keys(ops).map((key) => {
+    const op = ops[key];
+    return op[io] && upCam(op[io].shape);
+  }).filter(x => x);
+
 module.exports = (data) => {
-  const types = resolveTypes(data.shapes);
+  const types = resolveTypes(data.shapes, {
+    inputShapes: findIOShapes(data.operations, 'input'),
+    outputShapes: findIOShapes(data.operations, 'output'),
```

Comment:
Not all request and response types end with `Request` or `Response`. That was a false assumption